walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -46,7 +46,7 @@
      */
     if(request.checkForOptionalAttendees()){
       ArrayList<TimeRange> availableTimes = findAvailableMeetingTimes(events, request, /* optionalAttendees= */true);
-      if(!request.checkForAttendees() || !availableTimes.isEmpty()){
```

Comment:
Arg, sorry about this. The original version makes sense to me now. I was somehow thinking that this was an && rather than ||. I'm fine with keeping the version you had before or the new version, since they should produce the same output, whatever you think is better.